### DISCUSSION TITLE
Adding single samples to project

### DISCUSSION THREAD

**Discussion by naamaeb** [broadinstitute/seqr#4370] [General]
I have around 800 GATK called exome samples, that are not joint called. 
Is there a way I can add them to a single project - 
would I be able to load them into seqr even though they are not joint called?

**Comment by hanars** [comment]
seqr only accepts joint called data. You can find documentation on how to joint call your data for seqr here: https://storage.googleapis.com/seqr-reference-data/seqr-vcf-info.pdf
